java/arcs/core/util/ArcsStrictMode.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.util
+
+/**
+ * Global configuration for ArcsStrictMode flags.
+ */
+object ArcsStrictMode {
+    private var strictHandles_ = ArcsStrictModeProvider.strictHandles()
+
+    /**
+     * If true, [Handle] operations not executed within a [Scheduler] dispatcher
+     * will throw an error.
+     */
+    var strictHandles
+        get() = strictHandles_
+        internal set(value) {
+            strictHandles_ = value
+        }
```

Comment:
Can this be done without the backing field?

E.g.

```kotlin
var strictHandles: Boolean = ArcsStrictModeProvider.strictHandles()
    internal set
```

note, also since this isn't using a variable name that maps well to a boolean by name-alone (like `isX` or `useX`), it's a good idea to be explicit about the type.